test/Mesh/interpolation.jl

```diff
@@ -192,16 +187,13 @@ function run_cubed_sphere_interpolation_test()
                                             polynomialorder = polynomialorder,
                                             meshwarp = CLIMA.Mesh.Topologies.cubedshellwarp)
 
-    model = AtmosModel(SphericalOrientation(),
-                       NoReferenceState(),
-                       ConstantViscosityWithDivergence(FT(0)),
-                       DryModel(),
-                       NoPrecipitation(),
-                       NoRadiation(),
-                       NoSubsidence{FT}(),
-                       nothing,
-                       NoFluxBC(),
-                       setup)
+    model = AtmosModel{FT}(AtmosLESConfiguration;
```

Comment:
I realise this is just used as part of the setup, but does it make sense to allow an `AtmosLESConfiguration` to use `SphericalOrientation`?

Reply:
It seems like a `SphericalOrientation` is used in `test/DGmethods/Euler/acousticwave-1d-imex.jl`, I just used LES here to not change anything.

Reply:
Why not use GCM?

Reply:
I think I decided based on the orientation (this originally used a `FlatOrientation()`), but I guess this does make more sense in the context of a spherical orientation/gcm. Ultimately, I was trying to not change the flow control. Want me to try changing this to `GCM` with `SphericalOrientation`?